### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by subhangi2731**
fixes #39 . Modified the code and unit tests added

### CODE REVIEW

**Review by paramsiddharth — APPROVED** [review]
Good job!

File: _test_/users.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+const endpoint = 'https://rest-fetch.herokuapp.com/users'
```
Comment (paramsiddharth) [inline _test_/users.test.js:2]: Should this run the test on the deployed API or the local API? This needs to be confirmed.

Comment (adityabisoi) [reply _test_/users.test.js:2]: @subhangi2731 Please change the link to a local API. We can merge it then

**Comment by adityabisoi** [comment]
LGTM

**Pull request merged**